src/circuit.rs

```diff
@@ -745,6 +789,27 @@ impl plonk::Circuit<pallas::Base> for Circuit {
                     0,
                 )?;
 
+                region.assign_advice(
+                    || "split_flag",
+                    config.advices[8],
+                    0,
+                    || {
+                        self.split_flag
+                            .map(|split_flag| pallas::Base::from(split_flag as u64))
+                    },
+                )?;
+
+                nf_old
+                    .inner()
+                    .copy_advice(|| "nf_old", &mut region, config.advices[0], 1)?;
+                region.assign_advice_from_instance(
+                    || "nf_old pub",
+                    config.primary,
+                    NF_OLD,
+                    config.advices[1],
+                    1,
```

Comment:
why these values?

Reply:
Here we create a region and we assign some values in some cells.
In the first row (offset=0), only the cell in the last advice column (config.advices[9]) is available.
We have two options

    assign nf_old and nf_old_pub on the same row and thus, assign them in the second row (offset =1) because there is no enough available cell in the first row
    assign nf_old in the last advice cell available on the first row (cell: (config.advices[9], 0)) and assign nf_old_pub in the next row (cell: (config.advices[0], 1))
    We selected the second option which fills all advice cells in the first row (offset = 0) of the region.